css/excercise_one/part-c/css/excercise_one.css

```
@@ -0,0 +1,516 @@
+* {
+  color: black;
+  font-family: "Poppins", sans-serif;
+  text-decoration: none;
+}
+
+.pt-sm {
+  padding-top: 8px;
+}
+
+.pt-lg {
+  padding-top: 70px;
+}
+
+.pt-xl {
+  padding-top: 250px;
+}
+
+.pr-md {
+  padding-right: 40px;
+}
+
+.pr-lg {
+  padding-right: 70px;
+}
+
+.pb-sm {
+  padding-bottom: 8px;
+}
+
+.pb-md {
+  padding-bottom: 40px;
+}
+
+.pb-lg {
+  padding-bottom: 70px;
+}
+
+.pb-xl {
+  padding-bottom: 120px;
+}
+
+.pl-md {
+  padding-left: 40px;
+}
+
+.pl-lg {
+  padding-left: 70px;
+}
+
+.px-md {
+  padding-right: 40px;
+  padding-left: 40px;
+}
+
+.px-xl {
+  padding-right: 120px;
+  padding-left: 120px;
+}
+
+.py-sm {
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+.m-0 {
+  margin-top: 0px;
+  margin-right: 0px;
+  margin-bottom: 0px;
+  margin-left: 0px;
+}
+
+.mt-0 {
+  margin-top: 0px;
+}
+
+.mt-sm {
+  margin-top: 8px;
+}
+
+.mt-md {
+  margin-top: 20px;
+}
+
+.mt-lg {
+  margin-top: 70px;
+}
+
+.mt-xl {
+  margin-top: 110px;
+}
+
+.mt-xxl {
+  margin-top: 300px;
+}
+
+.mt-ng {
+  margin-top: -20px;
+  margin-right: 0px;
+  margin-bottom: 0px;
+  margin-left: 0px;
```

We can change this to be done in one line. Also when using 0, is common to avoid adding the px, %, rem, etc
```suggestion
  margin-top: -20px 0 0;
```